modules/gapi/include/opencv2/gapi/core.hpp

```diff
@@ -501,6 +504,42 @@ namespace core {
             return in.withType(in.depth, in.chan).withSize(dsize);
         }
     };
+
+    G_TYPED_KERNEL(GParseSSDWL, <GDetections(GMat, GOpaque<Size>, float, int)>, "org.opencv.core.parseSSDWL") {
+        static std::tuple<GArrayDesc,GArrayDesc> outMeta(const GMatDesc&, const GOpaqueDesc&, float, int) {
+            return std::make_tuple(empty_array_desc(), empty_array_desc());
+        }
+    };
+
+    G_TYPED_KERNEL(GParseSSD, <GRects(cv::GMat, cv::GOpaque<cv::Size>, float, bool)>, "org.opencv.core.parseSSD") {
+        static cv::GArrayDesc outMeta(const cv::GMatDesc&, const cv::GOpaqueDesc&, float, bool) {
+            return cv::empty_array_desc();
+        }
+    };
+
+    G_TYPED_KERNEL(GParseYolo, <GDetections(GMat, GOpaque<Size>, float, float, YoloAnchors)>, "org.opencv.core.parseYolo") {
+        static std::tuple<GArrayDesc, GArrayDesc> outMeta(const GMatDesc&, const GOpaqueDesc&, float, float, const YoloAnchors&) {
+            return std::make_tuple(empty_array_desc(), empty_array_desc());
+        }
+        static const YoloAnchors& defaultAnchors() {
+            static YoloAnchors anchors {
+                0.57273f, 0.677385f, 1.87446f, 2.06253f, 3.33843f, 5.47434f, 7.88282f, 3.52778f, 9.77052f, 9.16828f
+            };
```

Comment:
We should specify a paper or a dataset where these anchors come from.

Reply:
Added `@note` to `@brief` of kernel with link to [openvinotoolkit docs](https://docs.openvinotoolkit.org/latest/omz_models_intel_yolo_v2_tiny_vehicle_detection_0001_description_yolo_v2_tiny_vehicle_detection_0001.html#output) (yolo-v2-tiny-vehicle-detection-0001). Is it?

Reply:
That's ok